curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/64347464f78cd9209545f35c.md

```diff
@@ -22,7 +22,7 @@ assert.match(code, /letters\.forEach\(/);
 You should pass your `createLabel` function reference to the `.forEach()` method.
 
 ```js
-assert.match(code, /letters\.forEach\(\s*createLabel\s*\)/);
+assert.match(code, /letters\.forEach\(\s*letter\s*=>\s*createLabel\(\s*letter\s*\)\s*\)|letters\.forEach\(\s*createLabel\s*\)/);
 ```
```

Comment:
test doesn't pass for this block of code which is also correct 
```
letters.forEach((letter) => createLabel(letter))
```
update regex so that users can optionally add brackets around `letter` argument.

Reply:
updated the same